server/src/core/user/types.ts

```diff
@@ -13,6 +13,11 @@ export interface User {
   email?: string,
   name: string,
   isAnonymous?: boolean,
+  /**
+   * indicates whether the added beneficiary user 
+   * has been verified or not.
+   */
+  isVetted?: boolean
```

Comment:
`Vetted` does not mean verified. `Vetted` means that the beneficiary can receive money from any donor. There should be another field for the `beneficiaryStatus` which could be `pending` or `verified`.

When we add new beneficiaries through the admin CLI, they should be added as vetted. Maybe `users.addBeneficiary` should be renamed to `users.addVettedBeneficiary` to be clear. But their status should be `pending` by default.

When field officers find beneficiaries, they will be added to the platform as vetted. But we may need to do some additional follow-up to ensure that the information is correct and their case is legitimate. Once we've done this we'll go back to the admin CLI and mark the beneficiary as `verified`.